lib/autocomplete-manager.coffee

```diff
@@ -393,6 +399,11 @@ class AutocompleteManager
         # Activate on removal of a space, a non-whitespace character, or a bracket-matcher pair
         shouldActivate = oldText is ' ' or oldText.trim().length is 1 or oldText in @bracketMatcherPairs
 
+      # Suppress activation if the editorView has classes that match the suppression list
+      if shouldActivate and @editorView?.classList? and @suppressForClasses?.length
```

Comment:
Do you need the checks for `@editorView?.classList?` and `@suppressForClasses?.length`? Will this get in here if there is no editor? Will the editor ever _not_ have a classList? Will `@suppressForClasses` ever be null? I think you can remove both of them and just have `if shouldActivate`